static/src/javascripts/projects/common/modules/experiments/tests/remote-render-epic.js

```diff
@@ -0,0 +1,189 @@
+// @flow
+
+import {
+    makeEpicABTest,
+    setupOnView,
+    emitBeginEvent,
+    setupClickHandling,
+    emitInsertEvent,
+} from 'common/modules/commercial/contributions-utilities';
+import { epicButtonsTemplate } from 'common/modules/commercial/templates/acquisitions-epic-buttons';
+import fetch from 'lib/fetch';
+import fastdom from 'lib/fastdom-promise';
+import config from 'lib/config';
+
+const campaignId = 'gdnwb_copts_memco_remote_epic_test_api';
+
+const buildKeywordTags = page => {
+    const keywordIds = page.keywordIds.split(',');
+    const keywords = page.keywords.split(',');
+    return keywordIds.map((id, idx) => ({
+        id,
+        type: 'Keyword',
+        title: keywords[idx],
+    }));
+};
+
+const fetchRemoteEpic = payload => {
+    const api = 'https://contributions.guardianapis.com/epic';
+
+    return fetch(api, {
+        method: 'post',
+        headers: { 'Content-Type': 'application/json' },
+        body: payload,
+    });
+};
+
+const checkResponseOk = response => {
+    if (response.ok) {
+        return response;
+    }
+
+    throw new Error(
+        `Contributions fetch failed with response code: ${response.status}`
+    );
+};
+
+const decodeJson = response => response.json();
+
+const products = ['CONTRIBUTION', 'MEMBERSHIP_SUPPORTER'];
+
+const remoteRenderTest = {
+    id: 'RemoteRenderEpic',
+    campaignId,
+
+    highPriority: true,
+
+    start: '2020-01-01',
+    expiry: '2020-02-29',
+
+    author: 'Nicolas Long',
+    description:
+        'A/B test local vs remote render of default epic, to validate Slot Machine approach and work to date',
+    successMeasure: 'Conversion rate',
+    idealOutcome: 'No difference between control and variant',
+
+    audienceCriteria: 'All',
+    audience: 1,
```

Comment:
to pick up 10% of the audience, set this to `0.1`

Reply:
go big or go home

Reply:
ha thanks! 100% wouldn't have been so great 🤦‍♂